crates/pink/pink-extension/src/topic.rs

```diff
@@ -0,0 +1,56 @@
+//! The TopicsBuilder implementation is copied from ink-env since it is no longer exported by ink-env-3.0.0.
```

Comment:
Fine, but let's also reach out to the upstream and ask them to open a hole for us

Reply:
https://github.com/paritytech/ink/issues/1196